src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/ImplementIDisposableCorrectlyTests.cs

```diff
@@ -1781,8 +1768,7 @@ Inherits B
         MyBase.Finalize()
     End Sub
 End Class|]
-",
-            GetCA1063BasicFinalizeOverrideResultAt(11, 14, "C"));
+");
```

Comment:
It would, but as I mentioned before, that diagnostic is yet unimplemented. Once it is, this, and probably other tests will need to be updated.